### ISSUE TITLE
sass-lint:disable-block re-enables rules after sass-lint:disable

### ISSUE THREAD

**Issue by danfishgold** [sasstools/sass-lint#1291]
.sass-lint.yml:
```yaml
rules:
  border-zero: 2
```
sscce.sass:
```sass
// sass-lint:disable border-zero
.a
  // sass-lint:disable-block border-zero
  .b
    border: none

.c
  border: none
```

**Command**: `$ sass-lint sscce.sass -v`
**Expected result**: no errors due to `sass-lint:disable` on line 1
**Actual result**: error on line 8
```
sscce.sass
  8:11  error  A value of `none` is not allowed. `0` must be used  border-zero
```

I'm using Sass Lint version 1.13.1

My guess is that `disable-block` re-enables the rule at the end of the block, regardless of `disable`s on higher level blocks.
